sandpack-react/src/components/ReactDevTools/index.tsx

```diff
@@ -42,22 +43,24 @@ export const SandpackReactDevTools = ({
     });
   }, []);
 
+  const resolvedClientId = clientId
+    ? clientId
+    : Object.keys(sandpack.clients)?.[0] ?? generatedClientId.current;
```

Comment:
Went this route to guarantee that `resolvedClientId` is a string, since `clientId` and `Object.keys(sandpack.clients)?.[0]` could be `undefined`